### ISSUE TITLE
Itom 4.3dev: Timeout ERROR in Ximea and MSMediaFoundation Grabber

### ISSUE THREAD

**Issue by BBertschinger** [itom-project/plugins#49]
In Current Dev-Version 4.3dev:

When Grabber Plugin is created and Snapshot-Tool is opened an acquisiont for the data type: *.idc is not possible.
Snapshot worksperfectly fine for *.bmp and *.jpg images.
Neither Sequence nor single shot works for IDC.

The programm hangs up and timeout error is created:

Message reads: "Timeout while saving picture"
And message in console reads: "Itom-Application caught an unknoen exception from: (ito:PythonEngine) from event type 43 (MetaCall)

Many Thanks to Ömer for reporting.

**Comment by magro11** [comment]
I tried to reproduce this, however I did not have success.

I used a Qt5 debug version of itom, compiled from the current master branch.

At first, I tried the DummyGrabber plugin (8bit greyscale): Here, I could store to workspace, bmp, and idc. Both a single acquisition and a stack acquisition. Then I tested the MSMediaFoundation with my color webcam and it worked, too.

Can somebody try to reproduce the error with the grayscale DummyGrabber in order to check whether the color rgba32 object might cause the error? Else, is it possible to reproduce the error under debug built? I will also try a release version.
